### PR TITLE
U/yoachim/observatory overhead

### PR DESCRIPTION
**Pull request by yoachim**
Refactor readtime as an overhead. Add shutter cool-down time an another possible contributor to overhead.

### CODE REVIEW

**Comment by rhiannonlynne** [comment]
@smr456 Steve Ritz - we're adding the minimum 15s time between exposures here.

File: rubin_sim/scheduler/modelObservatory/kinem_model.py [hunk 1]
```diff
@@ -605,19 +610,35 @@ def slew_times(
...
+
     def observe(self, observation, mjd, rotTelPos=None, lax_dome=True):
         """observe a target, and return the slewtime and visit time for the action
 
         If slew is not allowed, returns np.nan and does not update state.
         """
+        if (observation["nexp"] > 2) & (
+            observation["exptime"] / observation["nexp"] < self.two_motion_time
+        ):
+            warnings.warn(
```
Comment (tribeiro) [inline rubin_sim/scheduler/modelObservatory/kinem_model.py:637]: I'd suggest adding a logger to the observatory model and using `self.log.warning` instead.. 

You could add something like this to the class, init method:

```
import logging

...
class Kinem_model(object):
    ...
    def __init__(
        self, location=None, park_alt=86.5, park_az=0.0, start_filter="r", mjd0=0, log=None
    ):

    self.log = log.getChild(type(self).__name__) if log is not None else logging.getLogger(type(self).__name__)
```

Comment (yoachim) [reply rubin_sim/scheduler/modelObservatory/kinem_model.py:637]: Looks like a logger would then print the warning for every single exposure. I like the `warnings.warn` behavior of only printing once.

File: rubin_sim/scheduler/modelObservatory/kinem_model.py [hunk 2]
```diff
@@ -104,6 +105,7 @@ def setup_camera(
         maxspeed=3.5,
         accel=1.0,
         decel=1.0,
+        two_motion_time=15.0,
```
[the remark this answers is not in the document]
Comment (yoachim) [reply rubin_sim/scheduler/modelObservatory/kinem_model.py:108]: Let's try `shutter_2motion_min_time`

**Pull request merged**